src/lib/logger.js

```diff
@@ -24,9 +24,17 @@ import type { LocaleType } from "../types";
 import { LOGGER_URL, AMPLITUDE_API_KEY } from "../config";
 
 export function getLogger(): LoggerType {
+  const { disableSetCookie } = window.xprops;
```

Comment:
```suggestion
  const { disableSetCookie = false} = Object(window.xprops);
```
@xuanwan05 actually, it looks like a few of our tests are failing due to `disableSetCookie` being undefined. 😅   I feel like it would be good to give this a default value anyway.  

We also may have to take into account the case where `window.xprops` is undefined.. I don't _think_ this will happen in real life, but its what's happening in the tests & its better to be safe than sorry.  😄   So that's why I proposed wrapping `window.xprops` in `Object()`.